components/ArrowPath/arrow.ts

```diff
@@ -0,0 +1,136 @@
+import {Layer} from '@deck.gl/core';
+// @ts-ignore
+import {Model, Geometry} from '@luma.gl/core';
+
+const defaultProps = {
+  getEnd: {type: 'accessor', value: (x: any) => x.path && x.path.length ? x.path.slice(-1)[0] : [0, 0]},
+  getColor: {type: 'accessor', value: [255, 255, 255, 255]},
+  getWidth: {type: 'accessor', value: 3},
+  getRotation: {type: 'accessor', value: 0},
+  widthUnits: 'pixels',
+  widthScale: {type: 'number', value: 1, min: 0},
+};
+
+const vs = `
```

Comment:
These two shaders can be put into their own file - have a look at how it's already been done for an example